Packages.props

```diff
@@ -18,9 +18,9 @@
 		<PackageReference Update="Microsoft.Azure.Management.Sql" Version="1.41.0-preview" />
 
 		<PackageReference Update="Microsoft.Data.SqlClient.AlwaysEncrypted.AzureKeyVaultProvider" Version="1.1.1" />
-		<PackageReference Update="Microsoft.Data.SqlClient" Version="2.1.3"/>
+		<PackageReference Update="Microsoft.Data.SqlClient" Version="3.0.0"/>
 		<PackageReference Update="Microsoft.SqlServer.SqlManagementObjects" Version="161.46367.54" />
-		<PackageReference Update="Microsoft.SqlServer.DACFx" Version="150.5290.2-preview" GeneratePathProperty="true" />
+		<PackageReference Update="Microsoft.SqlServer.DACFx" Version="160.5317.5-preview" GeneratePathProperty="true" />
```

Comment:
is this nuget already pushed to nuget.org? if not then we need to do that then you can use it here.

Reply:
Please make sure this process is documented somewhere (DacFX repo seems like a good place...) if it isn't already 😄

Reply:
we already have a doc for dacfx nuget. Sai please take a look and if it's not clear let me know